.gitignore

```diff
@@ -17,3 +17,5 @@ cluster.env
 kubedns
 # pilot
 pilot/platform/config
+# lint
+lintconfig.json
```

Comment:
instead, should we just generated the temporary file in some /tmp dir ? replacing it every time the lint.sh script runs.

Reply:
done